### ISSUE TITLE
关于计算p-net 损失

### ISSUE THREAD

**Issue by guojiapeng00** [AITTSMD/MTCNN-Tensorflow#76]
loss = -tf.log(label_prob+1e-10)   这个地方一直没想明白，不知为什么要加1e-10呀？谢谢

**Comment by AITTSMD** [comment]
@guojiapeng00 防止下溢出

**Comment by guojiapeng00** [comment]
请问您的意思是防止得 inf 吗
